vmngclient/api/software_action_api.py

```diff
@@ -123,7 +121,7 @@ def install(
         elif image_version and not image:
             version = cast(str, image_version)
         else:
-            raise VersionDeclarationError("You can not provide software_image and image version at the same time")
+            raise VersionDeclarationError("You can not provide image and image version at the same time")
         install_specification = get_install_specification(devices.first())
 
         url = "/dataservice/device/action/install"
```

Comment:
Can we implement endpoints for these in this PR already?
`/dataservice/device/action/changepartition`
`/dataservice/device/action/install`

Reply:
Next PR